yarpl/test/Observable_test.cpp

```diff
@@ -153,39 +143,34 @@ TEST(Observable, ItemsCollectedSynchronously) {
  * in a Vector which could then be consumed on another thread.
  */
 TEST(DISABLED_Observable, ItemsCollectedAsynchronously) {
```

Comment:
Is it possible to enable these now :)

Reply:
it should. Let me do it in the next diff.
thanks for the suggestion